### PR TITLE
Config option

### PR DESCRIPTION
**Pull request by bschommer**
This PR allows it to use relative paths in the .ini files for tools and libraries. The paths are relative to the given .ini file.

It also introduces a new option `-target` that allows it to select .ini files with the given name.

### CODE REVIEW

File: driver/Configuration.ml [hunk 1]
```diff
@@ -10,6 +10,7 @@
 (*                                                                     *)
 (* *********************************************************************)
 
+open Filename
```
Comment (xavierleroy) [inline driver/Configuration.ml]: Cosmetic: I prefer not to open Filename and qualify explicitly.  E.g. "Filename.concat" is more readable than "concat" (which could be some other kind of concatenation operation).

File: driver/Configuration.ml [hunk 2]
```diff
@@ -82,9 +94,19 @@ let get_config_list key =
...
   | vl -> vl
 
-let prepro = get_config_list "prepro"
-let asm = get_config_list "asm"
-let linker = get_config_list "linker"
+let tool_absolute_path tools =
+  match tools with
+  | [] -> []
+  | tool::args -> let tool =
+      if is_implicit tool && dirname tool = current_dir_name then
```
Comment (xavierleroy) [inline driver/Configuration.ml]: I had the impression that "Filename.is_implicit tool" suffices here.  In other words, isn't it the case that "dirname tool = current_dir_name" as soon as "is_implicit tool = true" ?

Comment (bschommer) [inline driver/Configuration.ml]: `Filename.is_implict "a/b"` returns true as well, but `Filename.dirname "a/b"` is equal to `a`.

**Comment by xavierleroy** [comment]
Looks fine to me.  Sorry for  the delay.  Merging!

**Pull request merged**